scripts/ufm_fabric_health/run_fabric_health_report.py

```diff
@@ -0,0 +1,323 @@
+import os
+import platform
+from http import HTTPStatus
+
+from utils.report_polling import ReportPolling
+
+try:
+    from utils.utils import Utils
+    from utils.ufm_rest_client import UfmRestClient, HTTPMethods
+    from utils.args_parser import ArgsParser
+    from utils.config_parser import ConfigParser
+    from utils.logger import Logger, LOG_LEVELS
+except ModuleNotFoundError as e:
+    if platform.system() == "Windows":
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'set PYTHONPATH=%PYTHONPATH%;{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}')
+    else:
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'export PYTHONPATH="${{PYTHONPATH}}:{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}"')
+
+
+class UfmFabricHealthConstants:
+    FABRIC_REPORT_API_URL = 'reports/Fabric_Health'
+    REPORTS_API_URL = 'reports'
+
+    UFM_FABRIC_SECTION_DUPLICATE_NODES = "duplicate_nodes"
+    UFM_FABRIC_SECTION_MAP_GUIDS_DESC = "map_guids_desc"
+
+    UFM_FABRIC_SECTION_CABLES = "cables"
+    UFM_FABRIC_SECTION_CABLES_ERRORS_ONLY = "cables_errors_only"
+
+    UFM_FABRIC_SECTION_DUPLICATE_ZERO_AND_LIDS = "duplicate_zero_and_lids"
+    UFM_FABRIC_SECTION_NON_OPT_LINKS = "non_opt_links"
+    UFM_FABRIC_SECTION_NON_OPT_SPEED_WIDTH = "non_opt_speed_width"
+    UFM_FABRIC_SECTION_LINK_SPEED = "link_speed"
+    UFM_FABRIC_SECTION_LINK_WIDTH = "link_width"
+    UFM_FABRIC_SECTION_EFFECTIVE_BER_CHECK = "effective_ber_check"
+    UFM_FABRIC_SECTION_SYMBOL_BER_CHECK = "symbol_ber_check"
+    UFM_FABRIC_SECTION_PHY_PORT_GRADE = "phy_port_grade"
+
+    UFM_FABRIC_SECTION_EYE_OPEN = "eye_open"
+    UFM_FABRIC_SECTION_MIN_BOUND = "min_bound"
+    UFM_FABRIC_SECTION_MAX_BOUND = "max_bound"
+    UFM_FABRIC_SECTION_EYE_OPEN_ERRORS_ONLY = "eye_open_errors_only"
+
+    UFM_FABRIC_SECTION_FIRMWARE = "firmware"
+
+    UFM_FABRIC_SECTION_SM_STATE = "sm_state"
+
+    UFM_FABRIC_SECTION_ALARMS = "ufm_alarms"
+
+    args_list = [
+        {
+            "name": f'--{UFM_FABRIC_SECTION_DUPLICATE_NODES}',
+            "help": "Duplicated Node Description"
```

Comment:
the description that you should add them to the readme file , it should be mentioned here as well
for all properties..